screens/HomeScreen/HomeScreen.js

```diff
@@ -76,12 +92,30 @@ function HomeScreen({ navigation }) {
           style={styles.image}
           source={require("../../assets/profile_icon.png")}
         />
-
         <Text style={styles.text}>Profile</Text>
       </TouchableOpacity>
-      <View style={styles.innerContainer}>
-        <GameFeed data={DATA} />
-      </View>
+
+        {/* RENDER LOCATION / FEED*/}
+        <View style={styles.innerContainer}>
+
+          <ShowContent number={mapView}/>
+          
+        </View>
+
+      {/* BOTTOM NAV BUTTONS */}
+      <View style={styles.row} >
+
+
+            <TouchableOpacity onPress={() => {setMapView(0)} }>
+                <Text style={styles.topText}> Map View </Text>
+            </TouchableOpacity>
+
```

Comment:
Just some styling notes on this nav bar - put a vertical line between the two buttons to make it closer to the Figma. Also, maybe you could dynamically change the styling of the button the is currently highlighted? Like make the text white and the background a darker orange? Something like the following:

<TouchableOpacity onPress={() => setMiddleView("MapScreen")}>
    <Text style={[styles.topText, middleView == "MapScreen" ? styles.highlighed : null]}
<TouchableOpacity />

You can apply multiple styles to the same component by putting all the styles you want to apply in an array.

Also, maybe make the nav bar take up more of the bottom part of the screen if possible.

Reply:
The way that I set up the buttons on the bottom is kinda fucked. I tried playing with it for almost 2 hours and got no where. If you wanted to take a look at it I think you could style better than me.